iina/MainMenuActions.swift

```diff
@@ -124,7 +124,7 @@ extension MainMenuActionHandler {
   }
 
   @objc func menuJumpTo(_ sender: NSMenuItem) {
-    Utility.quickPromptPanel("jump_to") { input in
+    Utility.quickPromptPanel("jump_to", inputValue: self.player.info.videoPosition?.stringRepresentationWithPrecision(3)) { input in
```

Comment:
3 sub-second digits is the highest precision selectable in the UI (and commonly used in video editing). I think it’s a good default. Alternatively, we could use the precision the user has previously chosen to be displayed in the player controls.

Reply:
3 digits is good IMO